scripts/validations.py

```diff
@@ -0,0 +1,125 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
```

Comment:
It currently gets set to a different `set` object in `parse_unlisted_addons`. If this was `unlisted_addons=UNLISTED_ADDONS` then I couldn't change what `UNLISTED_ADDONS` pointed to, so `parse_unlisted_addons` would need to mutate the `set` it was originally set to or `parse_unlisted_addons` would need to run before this function is defined and that file would be required to exist for this file to load. It just doesn't seem like a good idea to me.